src/App.jsx

```diff
@@ -1,5 +1,33 @@
+import { useEffect } from 'react';
+
+import { useDispatch, useSelector } from 'react-redux';
+import { loadRegions, loadCategories } from './actions';
```

Comment:
```suggestion
import { useEffect } from 'react';
import { useDispatch, useSelector } from 'react-redux';

import { loadRegions, loadCategories } from './actions';
```
이렇게 공백으로 외부/내부 모듈을 구분하는건 어떤가요?

Reply:
https://github.com/CodeSoom/react-week5-assignment-1/pull/59/commits/acf0a1e19416a134388f9d53c6fb978d96b8792f 에서 변경했습니다. 개발자마다 공백으로 의미를 나누는 방법은 다르겠지만 전 redux 모듈들, 컴포넌트 모듈들, 상수모듈들 등 이렇게 의미를 부여해서 나눠주고 있는데 이렇게 해주어도 괜찮을까요?

Reply:
기준이 있다면 괜찮을 것 같아요~ 관련해서 eslint rule을 설정해서 팀간에 컨벤션을 관리하기도 합니다. 참고해 보셔도 좋을 것 같네요

- https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/order.md

Reply:
감사합니다. 이런 룰이 있는진 몰랐네요!